librosa/core/spectrum.py

```diff
@@ -370,7 +370,7 @@ def stft(
     else:
         off_start = 0
 
-    n_columns = util.MAX_MEM_BLOCK // (np.prod(y_frames.shape[:-1]) * y_frames.itemsize)
+    n_columns = int(util.MAX_MEM_BLOCK // (np.prod(y_frames.shape[:-1]) * y_frames.itemsize))
```

Comment:
force cast here because mypy can't figure out that // means integer :roll_eyes:

Reply:
FYI if mypy understand the operands, it should be well aware of `//`. I guess the problem here is that the operand `np.prod` has ambiguous type (it can return `np.ndarray` as well). Pulling the `int` into just the `np.prod` would be sufficient, assuming `util.MAX_MEM_BLOCK` is an `int`.

Reply:
Yeah that's probably right, though I think it's actually clearer to push the cast all the way out.